src/map/utils/charutils.cpp

```diff
@@ -6616,26 +6616,23 @@ namespace charutils
             return false;
         }
 
-        if (entity->targid < 0x400 || entity->targid >= 0x800)
+        if (entity->objtype == TYPE_MOB)
         {
-            if (entity->objtype == TYPE_MOB)
-            {
-                spawnlist = &PChar->SpawnMOBList;
-            }
-            else if (entity->objtype == TYPE_NPC)
-            {
-                spawnlist = &PChar->SpawnNPCList;
-            }
+            spawnlist = &PChar->SpawnMOBList;
+        }
+        else if (entity->objtype == TYPE_NPC)
+        {
+            spawnlist = &PChar->SpawnNPCList;
         }
-        else if (entity->targid < 0x700)
+        else if (entity->targid == TYPE_PC)
```

Comment:
I think you meant to change these from `targid ` to `objtype ` like the one above

Reply:
Oops. fixed. thanks for catching that